postprocessing/janus-pp-rec.c

```diff
@@ -186,6 +190,10 @@ int main(int argc, char *argv[])
 		if(val >= 0)
 			video_orient_extmap_id = val;
 	}
+	char *extension = NULL;
+	if(args_info.format_given || (g_getenv("JANUS_PPREC_FORMAT") != NULL)) {
+		extension = g_strdup(args_info.format_given ? args_info.format_arg : g_getenv("JANUS_PPREC_FORMAT"));
```

Comment:
I don't see a `g_free` for the `g_strdup` above, which means this is introducing a leak. To be fair, though, I think we're doing the same already for other variables, in `janus-pp-rec.c`, so we can ignore this for now (also because we have different exit places).